docs/bin/plugin_formatter.py

```diff
@@ -185,7 +185,9 @@ def write_data(text, output_dir, outputname, module=None):
         fname = os.path.join(output_dir, outputname)
         fname = fname.replace(".py", "")
 
-        update_file_if_different(fname, to_bytes(text))
+        updated = update_file_if_different(fname, to_bytes(text))
```

Comment:
I think there's one disadvantage about not printing the filename first: if an uncaught exception occurs while handling the file, it might be unclear during processing of which file the exception occured.

How about adding a `try ... except` which catches exceptions, prints `rendering: ...` and re-raises the exception? Quite a hack, but should do the trick :)

Reply:
Good point, I considered adding a verbose mode but since most people would run this through `make` it wouldn't be very convenient :-/